lib/arena.rb

```diff
@@ -1,2 +1,79 @@
+require "pry"
+require_relative "./gladiator"
+
 class Arena
+  attr_accessor :name, :gladiators
+
+  def initialize name
+    @name = name.capitalize
+    @gladiators = []
+  end
+
+  def add_gladiator (gladiator)
+    unless gladiators.length == 2
+    gladiators << gladiator
+    end
+  end
+
+  def fight
+    if gladiators.length >= 2
+      if gladiators[0].name == "Maximus"
```

Comment:
what about breaking this series of checks down into helper methods, such as `maximus?` or utilize case statements